### ISSUE TITLE
Itemization of 1.5.0 seemingly off.

### ISSUE THREAD

**Issue by Jaypy** [elysium-project/itemization#17]
Hey, I was looking at the stat updates, for the Arcanist mage tier 1 set, coming with the 1.5.0 patch, in the sql files on here. First there are some simple values that seem right to me, but then there's a bunch of update queries and finding the Arcanist updates in there, it seems it just ignores any updates this patch for some reason. (https://github.com/elysium-project/itemization/blob/master/sql/1.05/1.05_pve_items.sql#L3356-L3365)

The original 1.5 stats stated in the top are correct according to this snapshot: 
https://web.archive.org/web/20050729100736/http://thottbot.com/?i=37308 (watch version at the bottom of the item, closest t1 snapshot i bothered to find).
also, 1.5.0 patch notes indicate tier sets getting upgraded. (http://wowwiki.wikia.com/wiki/Patch_1.5.0)

So basically common sense tells me something is wrong, but if there's something I missed, please enlighten me =).

**Comment by m-laniakea** [comment]
Did you clear your WDB folder?

**Comment by Jaypy** [comment]
1.5.0 is yet to be released. I'm looking at the current "itemization-to-be" in this repo..

**Comment by m-laniakea** [comment]
@Jaypy Right, but that wasn't  the issue you reported. 
It sounds like you applied the 1.05 patch here to your local server, but didn't see item stats being updated. If that was the case, delete your WDB folder.

**Comment by Jaypy** [comment]
Well, I dont think i said anything about applying anything to a local server. I simply said that I looked through the sql files on this repository and could deduce that the current updates to 1.5.0 would be "seemingly off".

I just wanted to make an official issue, to see if it is being looked at or if I'm getting something wrong(e.g. other ppl see the items updated correctly).

So this is really more of a "concern" than an issue. Apologies for the misunderstanding.
